acceptance/support/common_steps.rb

```diff
@@ -336,9 +337,10 @@ def when_I_want_to_select_component_properties(attribute, text)
   end
 
   def and_I_click_on_the_three_dots
-    sleep 1 # Arbitrary delay, possibly required due to focus issues
+    # sleep 1 # Arbitrary delay, possibly required due to focus issues
```

Comment:
can we remove this if we're not gonna sleep?

Reply:
Done